src/screens/Settings/SettingsPage.js

```diff
@@ -76,6 +77,15 @@ class SettingsPage extends Component {
     }
   };
 
+  handleLanguageChange = async (change) => {
+    // set active language with react-localize-redux (redux i18n library)
+    await this.props.setActiveLanguage(change);
+    // set active language in device (redux store)
+    await this.props.saveDeviceLanguage(change);
+    // save device settings to asyncstorage (device storage)
+    await deviceServices.set(this.props.device);
```

Comment:
Have we not set up device storage as where we store redux data yet? I thought we had in some previous work, but I could be misremembering. If we have done that, do we have to set both redux and device storage separately?

Reply:
We're saving to/fetching from device storage one item at a time, and whenever the user makes a change to a given item, that item has to be changed in both the Redux store and device storage. Not super efficient code-wise, but a) I figured if the Redux store gets big it wouldn't be efficient to save the whole thing to device storage every time the user makes a change, and b) If we want to make async calls in Redux actions, I believe we need to add redux-thunk or something. (But I just noticed that getEvents in EventsMap appears to work even though it's making an API call with an async dispatch function sans thunk.)

Reply:
Sure that's fine! I only ask because if another dev tries to build similar work, I could see them missing this and forgetting to save to one of the places. I'm just looking for ways to make it was easy as possible for future contributors to expand on our patterns. 

And yeah I don't think you need to use `redux-thunk` for async calls. I'm pretty sure that library just makes it easier to make API calls, removing some of the boilerplate. But it's not required.

Reply:
I do believe that we're already saving all of the Events to Redux already, and that's definitely going to be the vast majority of the size of the Redux store. Maybe we don't worry about whether the Redux store is too big until that actually happens, and so go for the easiest to follow code rather than the most optimized code at this stage?

It's fine to retain this approach, though, especially if we encapsulate saving to both of those places into some kind of utility so we don't have to remember to save to both every time.

Reply:
Ok, I'll do some refactoring of this in the next PR-- I agree that we need more utilities for hooking up Redux and the async store, and if I don't have to add another library just to handle async calls from Redux dispatch, so much the better.